DEV_IDE_SETUP.md

```diff
@@ -0,0 +1,66 @@
+# IntelliJ Development Instance Setup
+
+## Determine which JDK you're running, and where it is
+
+Inside IntelliJ, go to the SDK menu (`Cmd+;` or `Ctrl+;`).
+
+Go to `Project`, and look under `SDK` to find the JDK you intend to use
+to develop the plugin.
+
+Press `Edit`, and make a note of `JDK home path`, as well as the version of the JDK.
+
+## Download and Install the JetBrains Runtime SDK
+
+Head to [the JetBrains Runtime releases page](https://github.com/JetBrains/JetBrainsRuntime/releases)
+and pick a version of the `JBRSDK` that matches your OS, architecture, and JDK version.
+
+Install it appropriately:
+
+- If you've downloaded a tarfile, just note the directory where you untarred it.
+- If you're on macOS and have chosen a `.pkg` release, it will be installed under `/Library/Java/JavaVirtualMachines/jbrsdk-<version and os>/Contents`
+
+Back in IntelliJ, head to the SDK menu (`Cmd+;` or `Ctrl+;`), and go to `SDKs` in the sidebar.
+
+Add a new SDK by pressing the `+` button at the top.
+Pick `Add IntelliJ Platform Plugin SDK`, and select the path where you've just installed it.
```

Comment:
I'm not sure why but when I followed this step I get "The selected directory is not a valid home for IntelliJ Platform Plugin SDK", I tried downloading different versions of the JBRSDK but I get the same error, can you provide more details on which versions we should select?

Reply:
I ended up dropping this step, because it also failed for me and @jastice helped me through the more standard, less involved setup process. Updated the docs.